### DISCUSSION TITLE
PnP Search Results Tokens - {Today} - Without time value

### DISCUSSION THREAD

**Discussion by dmoorjmalani** [microsoft-search/pnp-modern-search#4033] [Q&A]
Hi - Right now  {Today} token resolves to Date and Time, is there anyway for it to not have time?

**Comment by kasperbolarsen** [comment]
Like getdate?
https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers

**Comment by dmoorjmalani** [comment]
Sorry, I was not clearer, i need it in the Query template
![image](https://github.com/user-attachments/assets/401b1e60-0d00-4266-9c19-dd1bd190a031)

**Reply by wobba** [reply]
Not without a change. Question? Why do you want to remove the time portion.. or what are you resolving? Remember that date fields store the date/time as Zero time (GMT).
